deployment/kubernetes/objects/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: objects
+description: A Helm chart for Objects API
+
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
+
+dependencies:
+  - name: postgresql
+    version: ~10.2.0
```

Comment:
Shouldn't we use a newer version?

Reply:
Our `INSTALL.rst` file requires Postgres 10 or above and our ci also used Postgres 10 to run the test suite.
Tbh I don't want to touch the helm chart again until we will stop supporting Postgres 10 for this project

Reply:
Ok.